src/components/Animal/Animal.js

```diff
@@ -1,3 +1,16 @@
-export default function Animal() {
-  return <></>;
+import './Animal.css';
+import useSound from 'use-sound';
+
+export default function Animal(props) {
+  const [play] = useSound(`${process.env.PUBLIC_URL}/animals/${props.type}.mp3`);
+
+  return (
+    <div className="animal" style={{ top: props.top, left: props.left }}>
+      <button onClick={play}>
+        <img alt={props.name} src={`${process.env.PUBLIC_URL}/animals/${props.type}.svg`} />
```

Comment:
You don't need the process.env.PUBLIC_URL.

`animals/${props.type}.svg` should work